main.go

```diff
@@ -327,6 +333,10 @@ func main() {
 	// However, doing so would affect either private ARC testers or current ARC users
 	// who run ARC without those variabls.
 	mgrPodName := os.Getenv("CONTROLLER_MANAGER_POD_NAME")
+	mgrPodNamespace := os.Getenv("CONTROLLER_MANAGER_POD_NAMESPACE")
+
+	listenerImage := os.Getenv("SCALE_SET_LISTENER_IMAGE") // default
```

Comment:
maybe we just go with `mgrPodImage := os.Getenv("CONTROLLER_MANAGER_POD_NAMESPACE_IMAGE")`?

Reply:
we should. 
A more general question is what would be the best way to pass the listener image and image pull secret into the controller? 
the current approach was added as a short-term solution. 😄